### PR TITLE
feat(addedDate): use addedDate

### PR DESCRIPTION
**Pull request by johnytiago**
Ticket: https://trello.com/c/ipMAHg9b/1160-migrate-research-output-original-publishing-date-to-added-date

### CODE REVIEW

**Review by peterstarling — APPROVED** [review]

File: packages/react-components/src/organisms/SharedResearchCard.tsx [hunk 1]
```diff
@@ -50,7 +50,14 @@ const typeStyles = css({
 
 type SharedResearchCardProps = Pick<
   ResearchOutputResponse,
-  'id' | 'created' | 'publishDate' | 'team' | 'title' | 'type' | 'link'
+  | 'id'
+  | 'created'
+  | 'publishDate'
```
Comment (jeysal) [inline packages/react-components/src/organisms/SharedResearchCard.tsx:55]:
```suggestion
```

and change to use added date in the code below

File: packages/react-components/src/templates/SharedResearchProposal.tsx [hunk 2]
```diff
@@ -43,14 +43,20 @@ const postedStyles = css({
 
 type SharedResearchProposalProps = Pick<
   ResearchOutputResponse,
-  'created' | 'publishDate' | 'team' | 'description' | 'title' | 'type'
+  | 'created'
+  | 'publishDate'
```
Comment (jeysal) [inline packages/react-components/src/templates/SharedResearchProposal.tsx:47]:
```suggestion
```

File: packages/react-components/src/templates/__tests__/SharedResearchList.test.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ const props: Omit<ComponentProps<typeof SharedResearchList>, 'children'> = {
       title: 'Output 1',
       type: 'Proposal',
       publishDate: new Date().toISOString(),
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/SharedResearchList.test.tsx:13]:
```suggestion
```

File: apps/asap-cli/package.json [hunk 4]
```diff
@@ -22,6 +22,7 @@
     "csv-parse": "^4.12.0",
     "got": "^11.6.0",
     "jwt-encode": "^1.0.1",
+    "luxon": "^1.26.0",
```
Comment (jeysal) [inline apps/asap-cli/package.json:25]: TIL the BE already uses luxon in some places. Bit unfortunate that we have an inconsistency here (FE uses date-fns) but no biggie

**Review by jeysal — CHANGES_REQUESTED** [review]
If you don't have the time to make the FE changes, or run into problem, let us know and we can help :)

File: packages/react-components/src/organisms/SharedResearchCard.tsx (hunk 1 above)
Comment (johnytiago) [reply packages/react-components/src/organisms/SharedResearchCard.tsx:55]: I guess I should only change this once the FE changes the copy

Comment (jeysal) [reply packages/react-components/src/organisms/SharedResearchCard.tsx:55]: Copy change is already happening in another PR

File: apps/asap-cli/package.json (hunk 4 above)
Comment (johnytiago) [reply apps/asap-cli/package.json:25]: Yeah we need to mock the time in some place. This makes it easier

**Review by jeysal — APPROVED** [review]
fe lgtm

**Pull request merged**